client/sites-dashboard/components/sites-grid.tsx

```diff
@@ -27,16 +27,31 @@ interface SitesGridProps {
 	className?: string;
 	isLoading: boolean;
 	sites: SiteExcerptData[];
+	siteSelectorMode?: boolean;
+	onSiteSelectBtnClick?: ( site: SiteExcerptData ) => void;
 }
 
-export const SitesGrid = ( { sites, isLoading, className }: SitesGridProps ) => {
+export const SitesGrid = ( props: SitesGridProps ) => {
+	const { sites, isLoading, className, siteSelectorMode = false, onSiteSelectBtnClick } = props;
+	const additionalProps = siteSelectorMode
+		? {
+				showLaunchNag: false,
+				showBadgeSection: false,
+				showThumbnailLink: false,
+				showSiteRenewLink: false,
```

Comment:
I was thinking the same before I decided to edit the existing one.

There is a big chance to get additional requirements from design to include some stuff that exists now, for example, a renewal link or something similar... To avoid future copy/paste, I would go with this approach.

Anyway, we didn't introduce any more complexity except toggling existing html blocks.

Reply:
Ok @bogiii your reasoning makes sense. I would suggest the same as Daniel: to invert the approach and go with generic components that can be more customizable.

I was thinking of creating a `<SitesGridItem />` component that accepts props, for example, `siteTitleRightAdornment`, instead of disabling specific sections of the grid item. But either approach works... let's not worry too much about it.